### PR TITLE
Changed type in wdl for target file and gatk jar.  Removed cromwell d…

### PR DESCRIPTION
**Pull request by LeeTL1220**
…ependency in docker file

Additional changes:
- Added docker build to Travis-CI, so that we will be alerted failures.

Trying to add a test for travis building docker.

Docker already installed in travis-ci

Fixing typo

Fixing cwd

Removing extraneous assemble command when doing a docker build

Adding CI=true to reduce log messages

Adding CI=true to reduce log messages

Don't generate a test report if we are testing the docker image.

Making bayesian het pulldown the new norm

### CODE REVIEW

File: scripts/wdl/case_gatk_acnv_workflow.wdl [hunk 1]
```diff
@@ -570,10 +571,40 @@ task HetPulldown {
     }
 }
 
+# Call heterozygous SNPs in the normal and then count the reads in the tumor for each position called position.
```
Comment (samuelklee) [inline scripts/wdl/case_gatk_acnv_workflow.wdl]: `each position called position` -> `each called position`

File: scripts/wdl/case_gatk_acnv_workflow.wdl [hunk 2]
```diff
@@ -570,10 +571,40 @@ task HetPulldown {
...
+# entity IDs can be the same value
```
Comment (samuelklee) [inline scripts/wdl/case_gatk_acnv_workflow.wdl]: capitalize entity?

**Comment by samuelklee** [comment]
Just a couple of things, otherwise looks good to go!

File: scripts/wdl/case_gatk_acnv_workflow.wdl (hunk 1 above)
Comment (LeeTL1220) [inline scripts/wdl/case_gatk_acnv_workflow.wdl]: Done

**Pull request merged**